wiki/iNiS_games/id.md

```diff
@@ -14,13 +14,13 @@ tags:
   - Ouendan
 ---
 
-# Gim dari iNiS
+# Permainan dari iNiS
 
 *Lihat juga: [osu! (disambiguasi)](/wiki/Disambiguation/osu!)*
 
-[Jenis permainan](/wiki/Game_mode) [osu!](/wiki/Game_mode/osu!) dan bahkan osu! sendiri berdasar pada gim ritme yang dikembangkan oleh **[iNiS](https://en.wikipedia.org/wiki/INiS)** untuk [Nintendo DS](https://id.wikipedia.org/wiki/Nintendo_DS) pada awal tahun 2000-an. ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) mengembangkan osu! untuk menjadi simulator dan editor level untuk gim - gim yang dikembangkan oleh iNiS, dan perlu diketahui anggota komunitas osu! paling awal saat itu pada umumnya juga merupakan penggemar gim tersebut. [Beatmap](/wiki/Beatmap) yang dibuat pada masa itu sangat dipengaruhi oleh desain level yang dibuat oleh iNiS.
+[Mode permainan](/wiki/Game_mode) [osu!](/wiki/Game_mode/osu!) dan osu! sendiri berasal dari *rhythm game* yang dikembangkan oleh **[iNiS](https://en.wikipedia.org/wiki/INiS)** untuk [Nintendo DS](https://id.wikipedia.org/wiki/Nintendo_DS) pada awal tahun 2000-an. ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) mengembangkan osu! untuk menjadi simulator dan editor level untuk permainan-permainan yang dikembangkan oleh iNiS, dan perlu diketahui anggota komunitas osu! paling awal saat itu pada umumnya juga merupakan penggemar permainan tersebut. [Beatmap](/wiki/Beatmap) yang dibuat saat itu sangat dipengaruhi oleh desain level yang dibuat oleh iNiS.
```

Comment:
`Nama "osu!" serta [mode permainan osu! itu sendiri](/wiki/Game_mode) berasal dari rangkaian *rhythm game* yang dikembangkan oleh **[iNiS](https://en.wikipedia.org/wiki/INiS)** untuk [Nintendo DS](https://id.wikipedia.org/wiki/Nintendo_DS) pada awal tahun 2000-an. Pada awal berdirinya osu!, ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) mengembangkan osu! sebagai suatu simulator dan editor level untuk permainan-permainan yang dikembangkan oleh iNiS. Berhubung para anggota komunitas osu! di kala itu pada umumnya juga merupakan penggemar permainan-permainan iNiS, [beatmap-beatmap](/wiki/Beatmap) yang ada pada saat itu juga sangat dipengaruhi oleh level-level yang dibuat oleh iNiS.`

mmmmm i know this paragraph doesn't exactly follow the structure of the original English paragraph line-to-line but i suppose this flows a looooooooooot better so yeah 🐇

Reply:
also i'm not sure about linking the wikipedia link to `https://id.wikipedia.org/wiki/Nintendo_DS` (instead of `https://ien.wikipedia.org/wiki/Nintendo_DS` tho, iirc i've never seen another article where the links in it are also "localized" per se

Reply:
Linking to translated wikipedia is usually done in translated article (see [Internet Relay Chat](https://github.com/ppy/osu-wiki/tree/master/wiki/Community/Internet_Relay_Chat) for example).